data/notifications.txt

```diff
@@ -1 +1,2 @@
+Test notification <a href="#" title="#">read more here</a>
```

Comment:
why having the same line twice?

Reply:
Just to print 2 notifications on screen in the demo - nothing code related